bcgov/biohubbc
# BHBC-2282: Fix draft load

(https://quartech.atlassian.net/browse/BHBC-2282)

Testing procedure:
Baseline:

Log into Dev
Throttle your network
Load a draft project
You will see an empty draft project
As a result of this code change:

Go to the app link for this PR
Log in
Throttle your network
Load a draft project
You will see the content.
